src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/domene/BarnehagebarnVisningDto.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.ks.sak.barnehagelister.domene
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class BarnehagebarnVisningDto(
+    val ident: String,
+    val fom: LocalDate,
+    val tom: LocalDate?,
+    val antallTimerBarnehage: Double,
+    val endringstype: String? = null,
+    val kommuneNavn: String,
+    val kommuneNr: String,
+    val fagsakId: Long? = null,
+    val fagsakstatus: String? = null,
+    val endretTid: LocalDateTime,
+    val avvik: Boolean? = null,
+)
+
+interface BarnehagebarnForListe {
```

Comment:
Dette må være et interface for at spring-paging skal fungere på en god måte.

Reply:
Synes det høres ut som en kommentar som kan være nyttig i koden